packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -390,14 +392,35 @@ const GroupForm = ({
               ))}
 
               {requirementSubForms.length < MAX_REQUIREMENTS && (
-                <CWButton
-                  type="button"
-                  label="Add requirement"
-                  iconLeft="plus"
-                  buttonWidth="full"
-                  buttonType="secondary"
-                  buttonHeight="med"
-                  onClick={addRequirementSubForm}
+                <CWTooltip
+                  content="Cannot add more than 10 requirements"
+                  placement="bottom"
+                  renderTrigger={(handleInteraction, isTooltipOpen) => (
+                    <CWButton
+                      type="button"
+                      label="Add requirement"
+                      iconLeft="plus"
+                      buttonWidth="full"
+                      buttonType="secondary"
+                      buttonHeight="med"
+                      onClick={addRequirementSubForm}
+                      disabled={requirementSubForms.length >= MAX_REQUIREMENTS}
+                      onMouseEnter={(e) => {
+                        if (requirementSubForms.length >= MAX_REQUIREMENTS) {
+                          handleMouseEnter({ e, handleInteraction });
+                        }
+                      }}
+                      onMouseLeave={(e) => {
+                        if (requirementSubForms.length >= MAX_REQUIREMENTS) {
+                          handleMouseLeave({
+                            e,
+                            isTooltipOpen,
+                            handleInteraction,
+                          });
```

Comment:
I think `handleMouseEnter`, `handleMouseLeave` are specific for menus. In this case, probably using `handleInteraction` instead of `handleMouseEnter`/`handleMouseLeave` would be enough. You need to check it